packages/conjure-client/src/httpApiBridge.ts

```diff
@@ -35,7 +35,7 @@ export interface IHttpEndpointOptions {
     requestMediaType?: MediaType;
 
     /** MIME type of the expected server response, often "application/json" or "application/octet-stream" */
-    responseMediaType?: MediaType;
+    responseMediaType?: string;
```

Comment:
Widening the enum here to any string might be a breaking change on `conjure-client`.

Looking at the alternative https://github.com/palantir/conjure-typescript-runtime/pull/170, do you have a preference between the two? I'm starting to lean towards `forceBinaryResponseBody`. Otherwise we're subtly signaling that `conjure-client` supports all response media types with this typing change.

Reply:
It looks like this is for the `Accept` header, so this change isn't strictly required for the `fetchBridge.ts` changes above.

Reply:
right - this change is to be able to pass arbitrary Accept headers

Reply:
I don't have a strong preference of this vs #170

Reply:
Let's go with this approach but without the `responseMediaType` change? Does that work for your use case?